### ISSUE TITLE
Support for managing MSK Configurations

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#8736]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The Managed Streaming for Kafka (MSK) service just recently added support for managing Kafka configurations: https://docs.aws.amazon.com/msk/latest/developerguide/msk-configuration.html

### New Resource

* `aws_msk_configuration`

### Potential Terraform Configuration

```hcl
resource "aws_msk_configuration" "example" {
  description    = ""
  kafka_versions = [""]
  name           = ""
  server_properties = ""
}
```

### References

* https://docs.aws.amazon.com/msk/latest/developerguide/msk-configuration.html
* AWS Go SDK v1.19.34

**Comment by bflad** [comment]
Pull request submitted: #8740

**Comment by bflad** [comment]
The new `aws_msk_configuration` resource has been merged and will release with version 2.12.0 of the Terraform AWS Provider, likely later today.

**Comment by bflad** [comment]
This has been released in [version 2.12.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2120-may-24-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
